pkgs/os-specific/linux/broadcom-sta/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation {
     (fetchpatch {
       name = "linux-4.8.patch";
       url = "https://aur.archlinux.org/cgit/aur.git/plain/004-linux48.patch?h=broadcom-wl-dkms";
```

Comment:
Yeah, lets avoid having to do this over and over again ;-) Thanks for the stable link!